notebooks/01_sampling.ipynb

```diff
@@ -0,0 +1,539 @@
+      "text/plain": [
+       "<Figure size 1200x400 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "distribution = pd.concat(\n",
+    "    [\n",
+    "        lidar_samples[[\"cancov\"]].assign(group=\"sample\"),\n",
+    "        pd.DataFrame({\"cancov\": filter(lambda v: v == v, cancov.ravel())}).assign(\n",
+    "            group=\"population\"\n",
+    "        ),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "distribution.hist(by=\"group\", bins=50, figsize=(12, 4));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need a deterministic method for uniquely identifying our sample points so that we can potentially extract additional data in the future. We'll use [Plus Codes](https://maps.google.com/pluscodes/), which encode latitude and longitude coordinates into short strings."
```

Comment:
Yeah, I've always thought they were cool but never had a use for them! I'm not sure how useful this *really* is since a sequential ID would have worked just as well so far, but it was a good excuse to bring them in.